### PR TITLE
[MNO-281] Add intercom

### PR DESCRIPTION
**Pull request by hedudelgado**

### CODE REVIEW

**Comment by hedudelgado** [comment]
https://github.com/maestrano/mno-enterprise/pull/71

**Comment by hedudelgado** [comment]
@ouranos PR Updated

File: src/app/index.run.coffee [hunk 1]
```diff
@@ -44,12 +44,24 @@ angular.module 'mnoEnterpriseAngular'
...
+  # Display flash messages from the backend in toastr
+  # They're passed this way:
+  #    ?flash={"msg":"An error message.","type":"error"}
   .run (toastr, $location) ->
     if flash = $location.search().flash
       message = JSON.parse(flash)
       toastr[message.type](message.msg, _.capitalize(message.type), timeout: 10000)
       $location.search('flash', null) # remove the flash from url
+
+  .run(($rootScope, AnalyticsSvc, INTERCOM_ID) ->
```
Comment (ouranos) [inline src/app/index.run.coffee]: `INTERCOM_ID` is not needed

**Comment by hedudelgado** [comment]
@x4d3  I applied the changes I spoke today with @ouranos .

File: src/app/index.controller.coffee [hunk 2]
```diff
@@ -1,6 +1,8 @@
 angular.module 'mnoEnterpriseAngular'
-  .controller 'IndexController', ($scope, $sce, GOOGLE_TAG_CONTAINER_ID) ->
+  .controller 'IndexController', ($scope, $sce, GOOGLE_TAG_CONTAINER_ID, INTERCOM_ID) ->
     'ngInject'
+    
+    $scope.intercom_id = INTERCOM_ID
```
Comment (x4d3) [inline src/app/index.controller.coffee]: COuld you pleae remove this ?

File: src/app/index.run.coffee [hunk 3]
```diff
@@ -44,12 +44,19 @@ angular.module 'mnoEnterpriseAngular'
 
     $translate.use(locale)
   )
-
-   # Display flash messages from the backend in toastr
-   # They're passed this way:
-   #    ?flash={"msg":"An error message.","type":"error"}
+  # Display flash messages from the backend in toastr
+  # They're passed this way:
+  #    ?flash={"msg":"An error message.","type":"error"}
```
Comment (ouranos) [inline src/app/index.run.coffee]: Still have the extra line here?

File: src/index.html [hunk 4]
```diff
@@ -1,6 +1,7 @@
 <!doctype html>
 <html ng-app="mnoEnterpriseAngular" ng-strict-di="true">
   <head>
+
```
Comment (ouranos) [inline src/index.html]: Remove trailing whitespace

**Comment by ouranos** [comment]
@hedudelgado great! Just see my last comments and we're good to merge.

File: src/app/index.run.coffee (hunk 3 above)
Comment (hedudelgado) [inline src/app/index.run.coffee]: @ouranos I am not quite sure what yo mean by removing the extra line, as the code is the same way it was added and I did not change it, see in the following PR, https://github.com/maestrano/mno-enterprise-angular/commit/bc44fc7a42a5d3f9f95b68fa27e3a8e2e00abee0

Check it the change I did worked for you please.

**Comment by ouranos** [comment]
Closing in favor of #55 

@hedudelgado have a look at this commit: https://github.com/maestrano/mno-enterprise-angular/pull/55/commits/ca1c871d68d0b7ed79a6a0569b708316e545c3b5

**Pull request closed without merging**